lib/ElementHandle.js

```diff
@@ -103,6 +103,19 @@ class ElementHandle {
     const objectId = this._remoteObject.objectId;
     return this._client.send('DOM.setFileInputFiles', { objectId, files });
   }
+
+  /**
+   * @return {!Object}
+   */
+  async boundingBox() {
+    const boxModel = await this._client.send('DOM.getBoxModel', { objectId: this._remoteObject.objectId });
```

Comment:
Values in `DOM.getBoxModel` are affected by the `box-sizing` css property. AFAIR they also include page scale (@wwwillchen might remember better).

`getBoundingClientRect` might work better for this use case.

> Would you recommend documenting this consideration?

Once this lands, https://github.com/GoogleChrome/puppeteer/pull/452 should work as a good-enough documentation.